test/ref/multinomial.cpp

```diff
@@ -24,7 +24,7 @@
 #include <migraphx/instruction.hpp>
 #include <migraphx/literal.hpp>
 #include <migraphx/make_op.hpp>
-#include <migraphx/onnx.hpp>
```

Comment:
I wonder how this even compiled before as the ref tests only link `migraphx` and `migraphx_ref`.